src/sentry/api/helpers/source_map_helper.py

```diff
@@ -59,7 +64,9 @@ def source_map_debug(project, event_id, exception_idx, frame_idx):
 
     urlparts = urlparse(abs_path)
 
-    if not (urlparts.scheme and urlparts.path):
+    # Node will produce file paths and not a fully-qualified URLs, so we don't need to check it.
+    # It's also guaranteed here that we will have a valid path, as we verify it few lines above.
+    if event.platform != "node" and not (urlparts.scheme and urlparts.path):
```

Comment:
Do we need to limit this to `node`? It could also be something like `edge` in the future.

Reply:
This platform check is all over the processing pipeline, so `edge` wouldnt be processed anyway now.

Reply:
Got it, I assume this is something we have to tackle all at once at some point then. (Also thinking about deno and stuff)

Reply:
Exactly this. We'll have to change it in other places first anyway, so we'll bump this here too.